src/ui/FormatSize/FormatSize.spec.tsx

```diff
@@ -0,0 +1,60 @@
+import { render, screen } from '../../testing';
+import { FormatSize } from './FormatSize';
+
+const METRIC_UNITS = ['B', 'KB', 'MB', 'GB', 'TB', 'PB'];
+const BINARY_UNITS = ['B', 'KiB', 'MiB', 'GiB', 'TiB', 'PiB'];
+
+describe('FormatSize', () => {
+  it('Should format to Bytes', () => {
+    render(<FormatSize prefix="metric">500</FormatSize>);
+    expect(screen.queryByText('500 B')).toBeInTheDocument();
+  });
+
+  it('Should format to KB', () => {
+    render(<FormatSize prefix="metric">1000</FormatSize>);
+    expect(screen.queryByText('1 KB')).toBeInTheDocument();
+  });
+
+  it('Should format to MB', () => {
+    render(<FormatSize prefix="metric">{1000 * 1000}</FormatSize>);
+    expect(screen.queryByText('1 MB')).toBeInTheDocument();
+  });
+
+  it('Should format to float number', () => {
+    render(<FormatSize prefix="metric">1500</FormatSize>);
+    expect(screen.queryByText('1.5 KB')).toBeInTheDocument();
+  });
+
+  it('Should format to PB', () => {
+    render(
+      <FormatSize prefix="metric" metricUnits={METRIC_UNITS}>
+        {100000000 * 10000000}
```

Comment:
Тут аналогично делаем:
```suggestion
        {10 ** 15}
```
И так же сделать со степенями двойки.